geomstats/special_orthogonal_group.py

```diff
@@ -85,7 +85,7 @@ def belongs(self, point, point_type=None):
         if point_type == 'vector':
             point = gs.to_ndarray(point, to_ndim=2)
             _, vec_dim = point.shape
-            return vec_dim == self.dimension
+            return [vec_dim == self.dimension]
```

Comment:
Can you return a gs.array here, for consistency with other parts of the code? Thanks.

Reply:
fixed, code now returns ```gs.array(vector_dim == self.dimension)```

Reply:
Nina, the travis-ci failures for the latest commit are due to ```gs.array(vector_dim == self.dimension)```. It's ok for ```[vector_dim == self.dimension]```. What would be a good fix here?

Reply:
Not sure, but maybe this can help: https://stackoverflow.com/questions/48571521/tensorflow-error-using-a-tf-tensor-as-a-python-bool-is-not-allowed

Otherwise just remove the assert.

Thanks! :)

Reply:
there are 3 cases of ```assert self.belongs(point, point_type)``` in special_orthogonal_group.py and 3 cases in special_euclidean_group.py. Should they all be removed? Setting this line as a debug point, these are the variables before the assertion:

```
In[5]: vec_dim == self.dimension
Out[5]: True
In[6]: gs.array(vec_dim == self.dimension)
Out[6]: <tf.Tensor 'Const_815:0' shape=() dtype=bool>
```

The gs.array() casts it to a Tensor, which cannot be used by the assert function as it's neither true/false.

Reply:
I've changed it so that it matches the corresponding function in special_euclidean_group.py, build is now passing